app/lib/personal_for_you.rb

```diff
@@ -110,11 +110,24 @@ def statuses_for_direct_follows(acct)
 
   # Get subscribed channels with full accounts
```

Comment:
This comment says 'subscribed' but it's 'enabled'

Reply:
(unless I'm confused?)

Reply:
yep. totally right. it _was_ subscribed, but no longer!